src/Prelude/Linear.hs

```diff
@@ -1,16 +1,13 @@
 {-# LANGUAGE LinearTypes #-}
 {-# LANGUAGE NoImplicitPrelude #-}
 
--- |
--- This module defines all the facilities you need when writing linear code. As
--- an analog to the original 'Prelude', it has the basic tools any haskell
--- program using @-XLinearHaskell@ will probably need.
---
--- This module is designed to be imported unqualifed.
+-- | This module redefines 'Prelude' data types, functions and type classes
```

Comment:
It doesn't “redfine data types”, so the sentence, as it stands, is a bit misleading. Doc is hard, I'll come back to it soon, and attempt a fix.